dts/arm/ti/am64x_main.dtsi

```diff
@@ -5,7 +5,7 @@
  */
 
 #include <freq.h>
-
+#include "am64x_power_domains.dtsi"
```

Comment:
Can we please call this file `am64x_main_...`? The AM64x has a MCU only power mode which turns off the entire main power main but keeps the MCU on. We should separate both to not accidentally manipulate in the wrong power domain.
```suggestion
#include "am64x_main_power_domains.dtsi"

```